rubi/rubi/log_test.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import requests
+from logging_config import get_logs
+
+logger = logging.getLogger(__name__)
+
+# Configure the logger
+logging.basicConfig(level=logging.DEBUG)
+
+# Log some events
+logger.debug("Debug message")
+logger.info("Informational message")
+logger.warning("Warning message")
+logger.error("Error message")
+
+# Retrieve the logs
+logs = get_logs()
+
+# Send logs to the Flask app API endpoint
+url = "http://localhost:5000/api/logs"  # Update the URL if necessary
+response = requests.post(url, json={"logs": logs})
+print(response.text)
```

Comment:
i like the idea of extending `logging.Handler` for use throughout the repo. i think we can provide this as an optional flag/setting somewhere that indicates the user would like to post logs to a `url` if desired. by default, lets have it configured to behave like a normal log statement. we could store the flask api (localhost:5000) in the `network_config` and allow the user to update this if they so choose.

another path here could be utilizing something like `OpenTelemetry ` #4. this may be more than we need right now, but from my understanding we can instrument the repo to take advantage of a lot of existing tooling. 

final thoughts here, we can take advantage of more granular logging as well to provide more clarity to what is going (`debug`, `warning`, etc.) in the repo as is today

Reply:
@ishandhanani I think creating a custom `logHandler` is a must for this. This means you don't have to go and add a whole bunch of code in order to log to this dashboard but can rather just add the custom log handler when you want to run the dash.

And then all you would need to do is have an if statement based on if you want to add the dash log handler and then everything else will work as intended (and the terminal logs will still be output in addition to the dash).

Reply:
On @denverbaumgartner's point another must is to go through the logs currently in the repo and make sure that all the logs being emitted are at a sensible level.